tasks/internal_reconcile_report_orphan/schemas/output.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/internal_reconcile_report_orphan/schemas/output.json",
+  "title": "internal_reconcile_report_orphan Lambda Output",
+  "description": "The output for the internal_reconcile_report_orphan Lambda.",
+  "type": "object",
+  "required": [
+    "jobId",
+    "anotherPage",
+    "orphans"
+  ],
+  "properties": {
+      "jobId": {
+          "type": "integer",
+          "description": "The unique ID of the reconciliation job."
+        },
+      "anotherPage": {
+        "description": "Indicates if more results can be retrieved on another page.",
+        "type": "boolean"
+      },
+      "orphans": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "required": [
+            "keyPath",
+            "Etag",
```

Comment:
s3Etag

Reply:
ok makes sense. I think we should be consistent with naming here.